src/scrabble.js

```diff
@@ -1,6 +1,111 @@
 class Scrabble {
-  // Write your implementation here
+  constructor (word) {
+    this.word = word
+    this.wordMultiply = word
+    this.scoreMap = {
```

Comment:
It doesn't look like your `scoreMap` ever changes. This could be placed outside of your constructor/outside of your class.